apps/alarm-cron/alarm-cron.yml

```diff
@@ -0,0 +1,17 @@
+name: Alarm Cron
+description: 'Run alarms, timers and stopwatches with countdowns and custom commands from the tray.'
+website: 'https://github.com/bl00mber/alarm-cron'
+repository: 'https://github.com/bl00mber/alarm-cron'
+keywords:
+    - alarm
+    - timer
+    - stopwatch
+    - cron
+    - clock
+    - time
+    - date
+    - countdown
+    - microsoft
```

Comment:
Not sure if `microsoft` is good and partially similar to other keywords, can you please clarify this.

Reply:
It means app also built for windows, shoud I remove it?